config/karma.config.js

```diff
@@ -1,6 +1,9 @@
 const webpackConfig = require('./webpack.config.karma.js');
 const path = require('path');
 
+// Allow crossplatform use of karma-chrome-launcher. See: https://github.com/karma-runner/karma-chrome-launcher#headless-chromium-with-puppeteer
```

Comment:
In the usage in that readme it has a 
`module.exports = function(config) {
  config.set({
    browsers: ['ChromeHeadless']
  })
}`
Do we need that export here?

Reply:
We have it, https://github.com/coveo/search-ui-extensions/blob/9ad796a129b0142c53ae5fbbb61b3a9b5d926294/config/karma.config.js#L50

We just defined the parameter sent to `config.set` as a variable instead of passing it directly.